src/main/kotlin/io/github/inductiveautomation/kindling/log/LogPanel.kt

```diff
@@ -268,6 +249,21 @@ class LogPanel(
 
     override val icon: Icon? = null
 
+    class Header(private val totalRows: Int) : JPanel(MigLayout("ins 0, fill")) {
```

Comment:
Inlined this class since it's so trivial now there's no settings logic to deal with